src/plugins/workspace/public/components/workspace_selector/workspace_selector.tsx

```diff
@@ -66,6 +66,7 @@ export const WorkspaceSelector = ({ coreStart, registeredUseCases$ }: Props) =>
   };
   const button = currentWorkspace ? (
     <EuiPanel
+      className="popoverButton"
```

Comment:
```suggestion
      className="workspaceSelectorPopoverButton"
```
Adding some prefix makes me feel safer, can we?

Reply:
Thanks for the advice, always updated.